### DISCUSSION TITLE
Weird issue with immich custom thumbnail path and rclone

### DISCUSSION THREAD

**Discussion by aaronjrodrigues** [immich-app/immich#13876] [Q&A]
I currently store all my uploads on a hetzner storage box which is mounted using rclone. I setup a custom thumbnail path so that all the thumbnails are stored locally on my server's ssd. However I've noticed that after a few hours, immich will go back to storing thumbnails on my hetzner storage box even though I've explicitly defined the thumbnails to be stored on my SSD.

For troubleshooting, I tried mounting the storage box using cifs in fstab and the issue doesn't exist anymore. I also tried mounting in rclone using cifs however the issue still exists, so it must be something about rclone.

Has anyone encountered this issue before, if so do you know of any possible solutions or workarounds? Any help would be appreciated.

I currently mount my storage box in rclone using this command:

```
sudo rclone mount gringotts: /home/server/gringotts \
   --allow-other \
   --vfs-cache-mode full \
   --vfs-cache-max-size 100G \
   --log-level INFO \
   --log-file=/home/server/logs/rclone.txt
```

**Comment by bo0tzz** [comment]
I don't know exactly what's going on here but rclone mounts are generally just a bit janky; I recommend avoiding them if you can.

**Comment by aaronjrodrigues** [comment]
The reason why I went for rclone mounts were because of vfs caching, do you know any alternatives to rclone which support vfs caching?

**Reply by bo0tzz** [reply]
Any mount worth its salt will have some caching. rclone just makes it a more explicit thing because rclone is a hack that wouldn't work right without it :P
